pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
 
-__Note:__ If the device is powered off or loses all network connectivity, it will remain in the `Online (Heartbeat Only)` state until the last target state fetch exceeds the device [API polling interval][poll-interval].
+__Note:__ If the device is powered off or loses all network connectivity, the `heartbeat` indictor show as connected (green) until the last target state fetch exceeds the device [API polling interval][poll-interval], at which time it will be marked as `Disconnected`.
 
 ## Debugging Device Status
 
 If you find your device to be displaying a status which is unclear even with the above notes, visit the [device debugging masterclass][debugging-masterclass] for more information.
 
+# Update Status
+
+The `Update status` and `Update status duration` fields on devices represent the state of the current application update to target state. 
+
+__Note:__ Data for these states will only get populated for devices with supervisor 16.7.0 or later.
+
+## Update statuses
+
+| Status                                       | Description                                                                                                                                                                      |
+|----------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Done**                              | The update, restart or configuration change is complete and the device is in its target state.                                          |
+| **Downloading**                       | The supervisor is downloading one or more container images associated with the target release                                           |
+| **Downloaded**                        | All container images in the target release have been downloaded. **Note:** If the device holds an update lock, it will stop in this state until the lock is released (depending on the [update strategy](https://docs.balena.io/learn/deploy/release-strategy/update-strategies/#controlling-the-update-strategy)). |
+| **Applying Changes**                  | The supervisor is performing actions to install the new release, i.e. killing old containers and making volumes/network/container changes.    |
+| **Rejected**                          | The target release has required services that do not meet the [necessary requirements](https://docs.balena.io/learn/develop/container-contracts/#container-contracts) to run. |
```

Comment:
It's a minor point, but the lack of the phrase 'necessary requirements' mentioned in the linked documentation isn't brilliant UX. I appreciate requirements are mentioned lots though, so this isn't as big a thing as the polling interval comment.